JuiceMaker/JuiceMaker/Controller/FruitStoreVC.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  FruitStoreVC.swift
+//  JuiceMaker
+//
+//  Created by hyunMac on 2023/09/15.
+//
+
+import UIKit
+
+class FruitStoreVC: UIViewController {
```

Comment:
[API Design Guidelines](https://www.swift.org/documentation/api-design-guidelines/#use-terminology-well)를 참고해볼까요?
이런 부분이 있네요:
**Avoid abbreviations.** Abbreviations, especially non-standard ones, are effectively terms-of-art, because understanding depends on correctly translating them into their non-abbreviated forms.
> The intended meaning for any abbreviation you use should be easily found by a web search.

Reply:
```FruitStoreVC``` -> ```FruitStoreViewController``` 로 변경하였습니다